cli/flox/src/utils/errors.rs

```diff
@@ -381,6 +381,9 @@ pub fn format_managed_error(err: &ManagedEnvironmentError) -> String {
             Please check the spelling of the remote environment
             and make sure that you have access to it.
         "},
+        ManagedEnvironmentError::UpstreamNotFound(env_ref, _floxhub) => formatdoc! {"
+            The environment {env_ref} does not exist.
```

Comment:
nonblocking: to me this seems like a message that should live on `UpstreamNotFound`, not here. We have a pretty printer defined elsewhere which is what we want to use any time we expect the user to encounter this error in pull. Any other time a user hits this that is pretty unlikely (e.g. their env.json is inaccurate), we don't need to special case, partially because we don't care that much, and partially because we don't have enough context here to print a truly pretty message (i.e. we don't know if we should add the "you can create it with" message)

Reply:
thanks, fixed, amended the error formatting and passing the error through in the printing method